src/static-site/home.html

```diff
@@ -56,7 +56,7 @@
     <div class="block block-2-content p-y-md p-x height-center">
         <div class="container">
             <div class="row p-t-0">
-                <div class="form-block col-md-6 p-b p-t p-x">
+                <div id="sign-up" class="form-block col-md-6 p-b p-t p-x">
```

Comment:
This page shouldn't be getting used anywhere anymore, but harmless edit.  Otherwise this looks good to me!